src/DeprecationMessageFactory.php

```diff
@@ -28,4 +28,12 @@ public static function createDeprecationWarningForSourceforgeUploadTask()
         return 'The usage of SourceforgeUploadTask is deprecated and will be removed in TYPO3 Surf Version 3.0.0';
     }
 
+    /**
+     * @return string
+     */
+    public static function createDeprecationWarningForTYPO3CopyConfigurationTask()
```

Comment:
What about passing the class name as string from the calling class to avoid multiple almost same functions?

Reply:
I don´t care about Code Duplication in this case. I would prefer a single responsibility method. So i can create different messages depending on the use case.